test/easyconfigs/easyconfigs.py

```diff
@@ -581,6 +581,8 @@ def check_dep_vars(self, gen, dep, dep_vars):
                                    r'CGmapTools-0\.1\.2', r'BatMeth2-2\.1'])],
             # NanoPlot, NanoComp use an older version of Seaborn
             'Seaborn': [(r'0\.10\.1', [r'NanoComp-1\.13\.1-', r'NanoPlot-1\.33\.0-'])],
+            # scenicplus uses a newer version of Rust
+            'Rust': [(r'1.65.0', [r'scenicplus-1\.0\.0'])],
```

Comment:
@deniskristak Please make this change in the pull request where it's required.

Also, isn't Rust just a *build* dependency for scenicplus?!